static/new_home/style.css

```diff
@@ -613,9 +613,8 @@ header
   border-radius: none;
 }
 
-.card{
-  border-radius: 5%;
-  height: 700px;
+.card-rounder{
+  border-radius: 1.2rem !important;
```

Comment:
I asked for a `border-radius` of `10px`, not `1.2rem` which looks too large.

Reply:
done !